src/config/main.js

```diff
@@ -7,6 +7,7 @@ import _ from "lodash";
 import { eventCodes } from "./trigger";
 import { init } from "@brown-ccv/behavioral-task-trials";
 import { getProlificId } from "../lib/utils";
+import app from './app-version'
```

Comment:
Hi @eldu :wave: Yeah, I thought about this too and I'd be open to taking a different approach.

My thinking with this approach is that the `app-version.json` is in `.gitignore` so for a fresh checkout it won't exist until it's created by `npm install`, and at that moment it will be in sync.  I think this covers the case of builds created by automation, like GitHub Actions, so at least that important case should work out OK.

I agree it's possible for these to get out of sync during local development, though, and there's nothing at the moment to prevent that.  I think it's a low-tech, best-effort approach similar to how Honeycomb is already handling Git version info in `git-version.js` (called `version.js` before this PR).  In both cases, I think automation should be OK, but local dev could get out of sync.

Do you think we should revisit this?  I was nervous about importing `package.json` directly as a whole, but maybe this is not as bad as I thought?

Reply:
Importing package.json doesn't seem too bad to me especially since this repo is public anyway? I'm asking around in CCV as well and we'll see what others think! 

Could also just import what you needed from the `package.json`
`import { name, version } from "package.json"`;

Reply:
OK, cool, thanks.

Thinking back, the only reason I changed this part of the code is that after the upgrade I started getting warnings like this:

> Should not import the named export 'version' (imported as 'version') from default-exporting module (only default export is available soon)

And then trying to follow up on that, it seemed like we might not want to import the whole thing, as in this [so](https://stackoverflow.com/questions/64993118/error-should-not-import-the-named-export-version-imported-as-version).

So I thought maybe another approach would be good.  But I'm also happy to revert this part and maybe open a separate issue to keep an eye on that warning?

Reply:
I'm wondering how this version parameter is actually intended to be used, as it seems a pretty complicated way to get the version number out. Is there a problem if it gets out of sync locally? perhaps storing alongside data collected from a study, to show which version of the code was used when collecting the data. Or is it not intended for that purpose? 

Is there something like this [so post](https://stackoverflow.com/a/22339262/5542581) which might be fit for purpose?

Reply:
Hi @hollandjg.  Yeah, I'm not sure how the parameter is used either, but it does seem to be stored along with task data, so that would be my guess as well.

I think the env vars approach might work.  If I understand it, we'd add some lines to each of our .env files like these:
```
REACT_APP_VERSION=$npm_package_version
REACT_APP_NAME=$npm_package_name
```
And this might let us access the values via `process.env`.  But I don't know if the `npm_` variables would be set when running as an Electron app, so we might need another approach in that case, so I'm not sure if this would simplify things overall.

For now, I think I will back this change out and go with named imports from package.json, like @eldu suggested and similar to what Honeycomb does already.  I'll open a separate ticket to capture the fact that with this upgrade, Webpack is starting to warn us about the current approach, and I'll add some of what we discussed here.

This might be worth tackling on its own, and I didn't really mean to make waves with this part of the PR! :smile: